src/test/resources/com/puppycrawl/tools/checkstyle/filters/strategy/FileLength/defaultSize.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/src/main/java/Checker/FileLength/Test.java b/src/main/java/Checker/FileLength/Test.java
```

Comment:
please fix path to files, it might not be critical in this particular case, but will be good practice ones we come to multifile  and different folders diffs.

Reply:
done